packages/ember-rdfa-editor/src/components/_private/link-rdfa-node-poc/modal.gts

```diff
@@ -62,19 +62,21 @@ type LinkRdfaNodeModalSig = {
     onSubmit: (body: SubmissionBody) => unknown;
     onCancel: () => unknown;
     predicateOptionGenerator: PredicateOptionGenerator;
-    subjectOptionGenerator: SubjectOptionGenerator;
+    subjectOptionGenerator: TargetOptionGenerator;
+    objectOptionGenerator: TargetOptionGenerator;
   };
 };
 
 export type TermOption<TermType extends SayTerm> = {
   label?: string;
   description?: string;
   term: TermType;
+  inverse?: boolean;
```

Comment:
This is not a good api, "inverse" doesn't make much sense here. It's the "inverse" of a backlink, which is already the inverse direction. 
Better to use a "direction" property with "incoming" | "outgoing"

It's probably even good to make it required, I don't think a default direction makes sense